tests/tests_cib_constraints_create.yml

```diff
@@ -563,7 +563,7 @@
           when:
             - '"pcmk.constraint.config.output-formats" in __test_pcs_capabilities'
           vars:
-            __test_expected_lines: >
+            __test_expected_lines: '
```

Comment:
I guess this will work - but the usual way to do a multiline string in yaml is to use a literal block scalar with `|`

Reply:
This needed few fixes in the following tasks, but I made it work. Thanks.